api_docs/message-formatting.md

```diff
@@ -0,0 +1,44 @@
+# Message formatting
+
+Zulip supports an extended version of Markdown for messages, as well as
+some HTML level special behavior. This article is currently a changelog
+for updates to these features.
+
+**Changes**: In Zulip 4.0 (feature level 24), the rarely used `!avatar()`
+and `!gravatar()` markup syntax, which was never documented and had an
+inconsistent syntax, were removed.
```

Comment:
Let's put this entry in a "Removed features" section at the bottom.

Reply:
Makes sense. Done in updated changes.